src/Wallabag/CoreBundle/Filter/EntryFilterType.php

```diff
@@ -24,7 +26,18 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                             'placeholder' => 'dd/mm/yyyy'),
                         'format' => 'dd/MM/yyyy',
                         'widget' => 'single_text'
-                    )));
+                )))
+            ->add('domainName', 'filter_text', array(
+                'apply_filter' => function (QueryInterface $filterQuery, $field, $values)
+                    {
+                        $value = $values['value'];
+                        if (strlen($value) <= 3 || empty($value)) {
```

Comment:
You're right ... but with LexikFormFilter, I don't believe we can make in another way.